src/react-apps/applications/service-development/src/features/deploy/components/deployPaper.tsx

```diff
@@ -0,0 +1,466 @@
+          <div
+            className={classNames({
+              [classes.checkIconPositionFix]: localRepoInSyncWithMaster === inSyncStatus.ready,
+            })}
+          >
+            <AltinnIcon
+              iconClass={classNames({
+                ['ai ai-check']: localRepoInSyncWithMaster === inSyncStatus.ready,
+                ['fa fa-info-circle']: localRepoInSyncWithMaster !== inSyncStatus.ready,
+              })}
+              iconColor={localRepoInSyncWithMaster === inSyncStatus.ready ?
+                theme.altinnPalette.primary.green : '#008FD6'}
+              padding='0px 0px 7px 0px'
+            />
+          </div>
+
+        </Grid>
+        <Grid item={true} xs={11}>
+          {renderInSyncText(localRepoInSyncWithMaster)}
+        </Grid>
+      </React.Fragment>
+    );
+  };
+
+  const renderInSyncText = (param: string) => {
+    switch (param) {
+
+      case inSyncStatus.ready:
+        return (
+          <Typography variant='h2' className={classes.listItemTitle}>
+            {getLanguageFromKey('deploy_to_test.shared_with_org_true', props.language)}
+          </Typography>
+        );
+
+      case inSyncStatus.ahead:
+        return (
+          <React.Fragment>
+            <Typography variant='h2' className={classes.listItemTitle}>
+              {getLanguageFromKey('deploy_to_test.shared_with_org_false', props.language)}
+            </Typography>
+            <Typography variant='body1'>
+              {getLanguageFromKey('deploy_to_test.shared_with_org_false_changes_will_not_be_visible_in_test_env',
+                props.language)}
+            </Typography>
+            <div style={{ marginTop: 20, marginBottom: 20 }}>
+              <VersionControlContainer
+                language={props.language}
+                type='shareButton'
+              />
+            </div>
+          </React.Fragment>
+        );
+
+      case inSyncStatus.behind:
+        return (
+          <React.Fragment>
+            <Typography variant='h2' className={classes.listItemTitle}>
+              {getLanguageFromKey('deploy_to_test.changes_made_by_others_in_your_organisation_title', props.language)}
+            </Typography>
+            <Typography variant='body1'>
+              {/* tslint:disable-next-line:max-line-length */}
+              {getLanguageFromKey('deploy_to_test.changes_made_by_others_in_your_organisation_is_not_visible_in_altinn_studio', props.language)}
+            </Typography>
+            <div style={{ marginTop: 20, marginBottom: 20 }}>
+              <VersionControlContainer
+                language={props.language}
+                type='fetchButton'
+              />
+            </div>
+          </React.Fragment>
+        );
+
+      default:
+        return null;
+    }
+  };
+
+  const renderCSharpCompilesText = (param: boolean) => {
+    const altinnWindow: IAltinnWindow = window as IAltinnWindow;
+    const { org, service } = altinnWindow;
```

Comment:
Tidier with deconstruction on the same line.
`const { org, service } = window as IAltinnWIndow;`

Reply:
Nice. Fixed.